src/Layout.scss

```diff
@@ -36,11 +48,11 @@
 }
 
 .nav-bar-component {
-  background-color: #3f51b2;
+  background-color: $black;
   height: 64px;
-  -webkit-box-shadow: 0px 2px rgba(0, 0, 0, 0.25);
+  /*-webkit-box-shadow: 0px 2px rgba(0, 0, 0, 0.25);
```

Comment:
Remove vendor specific prefixes, they are supposed to be added automatically when we compile the scss.